modules/utils.js

```diff
@@ -0,0 +1,48 @@
+const _ = require('lodash');
+
+function prepareDbObjectForClient(obj) {
+    const id = obj._id.toHexString();
+
+    return _(obj)
+        .omit('_id', '__v')
+        .assign({ id: id })
+        .value();
+}
+
+/**
+ * @param  {Function}     callback
+ * @param  {Object}       callback.err     error that will be send
+ * @param  {Number}       callback.body    body that will be send
+ * @param  {Object}       callback.options additional options
+ *
+ * @param  {Object}       options          additional options that will be handled by server
+ * @param  {Number}       options.status   status to send by server
+ * @param  {String}       options.location location to send by server
+ */
+function sendResponse(callback, options) {
+    return (err, result) => {
+        if (err) {
+            callback(err, void 1, options);
```

Comment:
I could have used undefined here true, I just feel a bit uncomfortable writing `undefined` ever since I understood it could be redefined, although I guess on server side code that could never happen?
And `null` will eventually send a body with `null` whilst `undefined` will keep the body empty